akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -1496,7 +1496,7 @@ private[akka] class ActorSystemImpl(
         case None =>
       }
       expiry match {
-        case Some(expired) if expired.isBefore(today) =>
+        case Some(expired) if expired.isBefore(today) || expired.isEqual(today) =>
```

Comment:
I don't understand this. Didn't you say that the key should be valid on the same day as the expiry date in the key?
If this check is true then we will `failExpiredLicenseKey`.

Example, today is Nov 7. A key with expiry Nov 7 should still be valid. With this change the dates will be equal and therefore fail.

Do we have some other date conversion mismatch earlier in the chain?

I'll debug with some real key.

Reply:
Doh - you are right. I was moving too quick between another area which was wrong. I'll fix it there and be done with it :)